code/modules/admin/verbs/adminhelp.dm

```diff
@@ -761,22 +819,28 @@ GLOBAL_DATUM_INIT(admin_help_ui_handler, /datum/admin_help_ui_handler, new)
 //what can be a client, ckey, or mob
 //log_in_blackbox: Whether or not this message with the blackbox system.
 //If disabled, this message should be logged with a different proc call
-/proc/admin_ticket_log(what, message, log_in_blackbox = TRUE)
+/proc/admin_ticket_log(what, message, log_in_blackbox = TRUE, player_message)
```

Comment:
doc this new arg! also to consider: should it be after `log_in_blackbox`?

Reply:
oops, probably not